recipes-bsp/imx-mkimage/imx-mkimage_git.bb

```diff
@@ -12,13 +12,16 @@ inherit native deploy
 
 CFLAGS = "-O2 -Wall -std=c99 -I ${STAGING_INCDIR} -L ${STAGING_LIBDIR}"
 
+REV_CHIP ?= "B0"
+REV_CHIP_mx8qxpc0 = "C0"
```

Comment:
In this case we can drop both patches for now and I'll work on them another time. When are we going to add the rest of the machines?
This would be the full change:
REV_CHIP ?= "B0"
REV_CHIP_imx8qxpc0mek = "C0"
REV_CHIP_imx8qxpc0lpddr4arm2 = "C0"

Reply:
Ok; please close it for now.